### PR TITLE
ipagroup: Fix changed flag, new test cases

### PR DESCRIPTION
**Pull request by t-woerner**
The changed flag returned by ipagroup calls have not been correct. This
change fixes this. Addtitionally new test cases have been added to make
sure that the changed flag is correct.

### CODE REVIEW

File: tests/group/test_group.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      ipaadmin_password: MyPassword123
+      name: group1
+      group:
+      - group2
+      - group3
+      action: member
+    register: result
+    failed_when: result.changed
+
+  - name: Ensure group3 ia present in group group1
```
Comment (rjeffman) [inline tests/group/test_group.yml]: "is present" not "ia present".

File: tests/group/test_group.yml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      name: group1
+      user:
+      - user1
+      - user2
+      - user3
+      action: member
+    register: result
+    failed_when: result.changed
+
+  #- ipagroup:
```
Comment (rjeffman) [inline tests/group/test_group.yml]: Left over? A comment? I didn't uderstand why this comment is here.

File: tests/group/test_group.yml (hunk 1 above)
Comment (t-woerner) [reply tests/group/test_group.yml]: Fixed

File: tests/group/test_group.yml (hunk 2 above)
Comment (t-woerner) [reply tests/group/test_group.yml]: I removed this section.

**Pull request merged**